blead/patches/001.patch

```diff
@@ -255,31 +270,35 @@ index b1da8e3..9fe743e 100644
      }
      return io;
    #else
-diff --git a/iperlsys.h b/iperlsys.h
-index b922af0..daed107 100644
---- a/iperlsys.h
-+++ b/iperlsys.h
-@@ -777,21 +777,8 @@ struct IPerlLIOInfo
- #    define PerlLIO_lstat(name, buf)    PerlLIO_stat((name), (buf))
- #  endif
- #  define PerlLIO_mktemp(file)          mktemp((file))
--#  if defined(OEMVS)
--#    if (__CHARSET_LIB == 1)
--    int asciiopen(const char* path, int oflag);
--    int asciiopen3(const char* path, int oflag, int perm);
--
--#      define PerlLIO_open(file, flag)          asciiopen((file), (flag))
--#      define PerlLIO_open3(file, flag, perm)   asciiopen3((file), (flag), (perm))
--#    else
--#      define PerlLIO_open(file, flag)          open((file), (flag))
--#      define PerlLIO_open3(file, flag, perm)   open((file), (flag), (perm))
--#    endif
--#  else
--#    define PerlLIO_open(file, flag)            open((file), (flag))
--#    define PerlLIO_open3(file, flag, perm)     open((file), (flag), (perm))
--#  endif
-+#  define PerlLIO_open(file, flag)            open((file), (flag))
-+#  define PerlLIO_open3(file, flag, perm)     open((file), (flag), (perm))
- #  define PerlLIO_read(fd, buf, count)  read((fd), (buf), (count))
- #  define PerlLIO_rename(old, new)      rename((old), (new))
- #  define PerlLIO_setmode(fd, mode)     setmode((fd), (mode))
+diff --git a/Configure b/Configure
+index bd96249..28f19c0 100755
+--- a/Configure
++++ b/Configure
+@@ -4566,7 +4566,7 @@ echo "Checking for GNU cc in disguise and/or its version number..." >&4
+ $cat >try.c <<EOM
+ #include <stdio.h>
+ int main() {
+-#if defined(__GNUC__) && !defined(__INTEL_COMPILER)
++#if defined(__GNUC__) && !defined(__INTEL_COMPILER) && !defined(__MVS__)
+ #ifdef __VERSION__
+ 	printf("%s\n", __VERSION__);
+ #else
+@@ -8726,10 +8726,14 @@ case "$useshrplib" in
+ true)
+ 	case "$userelocatableinc" in
+ 	true|define)
+-		echo "Cannot build with both -Duserelocatableinc and -Duseshrplib" >&4
```

Comment:
why did you pull this check?

Reply:
I adjusted it below so that it applies only to non-z/OS since we don't use rpath.

Reply:
I'll add a comment to clarify

Reply:
Added echo statement below